### DISCUSSION TITLE
Still Waiting for GitHub Education Pack Approval – What Should I Do?

### DISCUSSION THREAD

**Discussion by ismaildasci** [community/community#142526] [GitHub Education]
### Hi everyone,

Hi everyone,

I applied for the GitHub Education Student Developer Pack, and I received a message stating:

“We’re still processing your application. Once this is complete you will have access to the student developer pack offers.”

It has been I haven’t received any further updates. I understand that the process may take some time, but I would like to know if there is anything I need to do on my end, or if I should continue waiting. Is this a typical waiting period, or should I contact support?

Additionally, if anyone has faced a similar situation, could you share your experience regarding how long the approval process took for you?

Thanks in advance for your help!

**Comment by TonyEinstein** [comment]
I have also encountered this problem. Would you please solve it now? How is it solved?

**Comment by Azzybana** [comment]
January is the start of a new year, if you fully read the manual, docs, whatever, you'll note that it may take extra time if you're in the group who signs up at semester changeover. I was approved on 1/21. The typical resume school date is the second Monday of January. That was the 13th. As of today, it's been less than 3 weeks since the spring semester rolled out, on average, with some lagging. 

It's been about 2 weeks since I signed up, due to issues with my sign up process, it's only been about a week since approval (I telecommute from FL to CA for school). My approval says 1/21. However, the manual says it can take weeks. 

The 'let people know at 5 days' came with a disclaimer, that's an optimistic goal, outside the start of a new semester. 

Expressing concern is valid. I have my own. I cancelled a pro trial I used to determine if the hassle of the runaround for student was worth it, and without some of the tools I've already grown accustomed to. It's a great reminder to not rely on them. Delays are delays, they happen. 

While it certainly never hurts to chime in earlier, sometimes there's not a lot of good to be done about it. As someone in school pursuing psychology, who's exiting tech, the first thing you need to learn about tech is patience. No matter how much you optimized, and believe me, I pull out all the stops, there's going to be twiddling your thumbs and waiting for the process to complete. If you'd read the WHOLE manual, you might have noticed. ;)

Being concerned is valid, being able to do anything about it, other than wait, during their busy time, is beyond most people's control, because there's a WHOLE LOT of students signing up, for the same reason you are. 

Take some deep breaths, relax, if you MUST have access to a language model, go look up the g4f module of python. Until then, patience. Eventually, you too, will enjoy the odd one-off anymore where it's not just trim, and you can sit and drool watching defrag work on a platter, for the first time in forever. It's fun. It really is. 

I miss that. It taught patience. 

Anyways, cheers!

- some other random student (and yes, I'm a furry who LOVES to panic in production as a feature)
